package.json

```diff
@@ -27,9 +27,9 @@
     "nx": "nx",
     "start": "npm-run-all --parallel \"ng serve cookbook\" \"build:core:watch\"",
     "start:flows": "ng serve flows",
-    "build": "ng build",
+    "build": "ng build cookbook",
```

Comment:
I believe this is a major blunder on my part when I originally did this, the build command should build the package (designsystem) not cookbook as far as I am concerned.

Reply:
ok, will fix